storage/rocksdb/rdb_i_s.cc

```diff
@@ -771,6 +772,71 @@ static ST_FIELD_INFO rdb_i_s_global_info_fields_info[] =
   ROCKSDB_FIELD_INFO_END
 };
 
+/*
+  Support for INFORMATION_SCHEMA.ROCKSDB_COMPACTION_STATS dynamic table
+ */
+static int rdb_i_s_compact_stats_fill_table(
+    my_core::THD *thd,
+    my_core::TABLE_LIST *tables,
+    my_core::Item *cond __attribute__((__unused__)))
+{
+  DBUG_ASSERT(thd != nullptr);
+  DBUG_ASSERT(tables != nullptr);
+
+  DBUG_ENTER("rdb_i_s_global_compact_stats_table");
+
+  int ret= 0;
+
+  rocksdb::DB *rdb= rdb_get_rocksdb_db();
+  Rdb_cf_manager& cf_manager= rdb_get_cf_manager();
+  DBUG_ASSERT(rdb != nullptr);
+
+  for (auto cf_name : cf_manager.get_cf_names())
+  {
+    rocksdb::ColumnFamilyHandle* cfh;
+    bool is_automatic;
+    /*
+       Only the cf name is important. Whether it was generated automatically
+       does not matter, so is_automatic is ignored.
+       */
+    cfh= cf_manager.get_cf(cf_name.c_str(), "", nullptr, &is_automatic);
+    if (cfh == nullptr)
```

Comment:
Add { } around the single line body to prevent problems in the future. Not all if's in the file do this, but we've been moving in this direction.